metron-deployment/inventory/full-dev-platform/group_vars/all

```diff
@@ -15,21 +15,38 @@
 #  limitations under the License.
 #
 
-#Ambari variables
+# which services should be started?
+services_to_start:
+  - mysql
+  - elasticsearch
+  - pcap-service
+  - metron-ui
+  - yaf
+  - snort
+  - snort-logs
+  - bro
+  - pcap-replay
+  - yaf-parser
+  - bro-parser
+  - snort-parser
+  - enrichment
```

Comment:
Yeah, I thought that as soon as we needed to start separate services, we'd have to separate the inventories. Was just about to create a duplicate directory and decided against it.